auto3dseg/algorithm_templates/dints/scripts/dummy_runner.py

```diff
@@ -0,0 +1,174 @@
+#!/usr/bin/env python
+
+import fire
+import numpy as np
+import os
+import torch
+import yaml
+
+from monai.bundle import ConfigParser
+from monai.inferers import sliding_window_inference
+from torch.cuda.amp import GradScaler, autocast
+
+
+class DummyRunnerDiNTS(object):
+    def __init__(self, output_path):
+        config_file = []
+        config_file.append(
+            os.path.join(output_path, "configs", "hyper_parameters.yaml")
+        )
+        config_file.append(os.path.join(output_path, "configs", "network.yaml"))
+        config_file.append(
+            os.path.join(output_path, "configs", "transforms_train.yaml")
+        )
+        config_file.append(
+            os.path.join(output_path, "configs", "transforms_validate.yaml")
+        )
+
+        parser = ConfigParser()
+        parser.read_config(config_file)
+
+        self.device = torch.device("cuda:0")
+        torch.cuda.set_device(self.device)
+
+        self.input_channels = parser.get_parsed_content("training#input_channels")
+        self.patch_size = parser.get_parsed_content("training#patch_size")
+        self.patch_size_valid = parser.get_parsed_content("training#patch_size_valid")
+        self.overlap_ratio = parser.get_parsed_content("training#overlap_ratio")
+
+        output_classes = parser.get_parsed_content("training#output_classes")
+        softmax = parser.get_parsed_content("training#softmax")
+        self.label_channels = 1 if softmax else output_classes
+
+        print("patch_size", self.patch_size)
+        print("patch_size_valid", self.patch_size_valid)
+        print("label_channels", self.label_channels)
+
+        self.model = parser.get_parsed_content("training_network#network")
+        self.model = self.model.to(self.device)
+
+        self.loss_function = parser.get_parsed_content("training#loss")
+
+        optimizer_part = parser.get_parsed_content("training#optimizer", instantiate=False)
+        self.optimizer = optimizer_part.instantiate(params=self.model.parameters())
+
+        train_transforms = parser.get_parsed_content("transforms_train")
+
+        data_stats_file = os.path.join(output_path, "..", "datastats.yaml")
```

Comment:
This seems a hard-coded path and could be changed if the user saves `datastats` in a different filename or path.